sentry-android-timber/src/test/java/io/sentry/core/SentryEventKtx.kt

```diff
@@ -1,8 +1,10 @@
-package io.sentry
+package io.sentry.core
+
+import io.sentry.SentryEvent
 
 /**
  * package-private hack.
```

Comment:
I'm confused 😅 is this hack not needed then, since we specify a different package now for this ext function? I think `throwable` is public now btw